docs/admin/workstation_architecture.rst

```diff
@@ -0,0 +1,55 @@
+SecureDrop Workstation Architecture
+===================================
+
+.. include:: ../includes/top-warning.rst
+
+.. _Networking Architecture:
+
+SecureDrop Workstation networking architecture
+----------------------------------------------
+One key security feature of Qubes OS is that it enables users to configure the
+appropriate level of network access for each VM. For example, you could have a
+VM for password storage that has no network access, a work VM that is firewalled
+to only connect to work servers, and a personal VM that always uses Tor.
+
+In the context of SecureDrop Workstation, these capabilities are used to
+minimize the risk that an adversary who is able to exploit a security
```

Comment:
nit: "minimize the risk _of_ an adversary who [...]" or "minimize the risk that an adversary who [...] _can_ exfiltrate documents"

Reply:
Agreed, that was also very inaccessible writing, let me know what you think of the new wording.

Reply:
LGTM!